intro-HPC/ch_preparing_the_environment.tex

```diff
@@ -917,6 +918,7 @@ \subsection{Organisation of modules in toolchains}
 interconnect features that require a vendor-specific MPI-implementation.
 
 \subsection{Activating and de-activating modules}
+\label{subsec:activating-and-deactivating-modules}
```

Comment:
@redfast00 Please make a dedicated section for `purge`, it's now buried way too deep in stuff related to `unload` (which is a lot less used imho)